src/pages/History/History.jsx

```diff
@@ -1,9 +1,51 @@
-import React from 'react'
+import { HorizontalVideoCard, RedirectToLogin, TopNav } from "../../components";
+import { useUserDetails } from "../../contexts";
+import { checkLogin } from "../../utils";
+import { clearHistory, removeFromHistory } from "../../utils/handleHistory";
+import styles from "./History.module.css";
```

Comment:
You can try using an index file and cleaning the imports to make it neater later on.